.github/workflows/verify.yml

```diff
@@ -56,49 +56,24 @@ jobs:
         path: builddir/meson-dist/*.tar.gz
 
   #
-  # Check source-format using ./scripts/source-format.sh
+  # Check source-format using pre-commit
   #
   source-format-check:
     needs: source-archive
     runs-on: ubuntu-latest
     if: (! contains(github.ref, 'ci')) || contains(github.ref, 'ci-source-format-check')
 
-    container: debian:bullseye
+    container: refenv/xnvme-devtools-ci:latest
 
     steps:
-    - name: Retrieve the source-archive
-      uses: actions/download-artifact@v2
-      with:
-        name: archive-src
-    - name: Extract the full-source-archive
-      run: |
-        tar xzf xnvme*.tar.gz --strip 1
-        rm xnvme*.tar.gz
-
-    - name: Install build-requirements
+    - uses: actions/checkout@v2
+    - name: Install requirements
       run: |
-        source scripts/pkgs/debian-bullseye.sh
-        apt -qy install wget gnupg2 lsb-release software-properties-common
-        wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -
-        add-apt-repository -y "deb http://apt.llvm.org/bullseye/ llvm-toolchain-bullseye-14 main"
-        apt -qy update
-        apt -qy install clang-format-14
-        update-alternatives --install /usr/bin/clang-format clang-format /usr/bin/clang-format-14 100
-
-    - name: xNVMe, run the source-format script
+        apt-get -qy update && apt-get -qy install shellcheck
+        pip3 install virtualenv==20.0.33
```

Comment:
Shouldn't this be in `refenv/xnvme-devtools-ci:latest`?

Reply:
Nice, catch, yes. Just added it here until it "worked". Fixing...